### ISSUE TITLE
'lotus-miner sectors update-state' to invalid state

### ISSUE THREAD

**Issue by Meatball13** [filecoin-project/lotus#3658]
**Describe the bug**
Note: This may have already been fixed, as this happened to a sector I had problems with a while back, but I wanted to pop it up just in case.  It did not appear that 'lotus-miner sectors update-state' has any checking to ensure only proper states are input.  

**To Reproduce**
1. Run lotus-miner sectors update-state --really-do-it ## <RandomWord>
2. State will be reset, but then the sector can no longer be properly removed or updated and will throw a 'ERROR: normal shutdown of state machine'

**Expected behavior**
If an invalid state is input it should fail to update the state, also, there should be a way to list the valid states.

**Screenshots**

**Version (run `lotus version`):**
Happened in 0.5.6'ish timeframe, but still cannot interract/update state as of 0.6.1

**Additional context**
I had a sector that was stuck looping through PC1 and bouncing between workers, so I tried to stop it by updating state and accidentally put in 'StartPacking' as the state.

`15: StartPacking sSet: NO  active: NO  tktH: 0     seedH: 0     deals: [64613 0 0 0 0 0 0 0 0 0 0 65173 65380 65717 0 0 0 0 0 0 0 0 0 65895 66202 66364 0 0 0 0 0 0 0 0 0 0 66662 0 0 0 0 0 0]                                                              toUpgrade:false`

While it stopped the sector from bouncing between the workers (good thing), I'm not able to try to update or remove the sector anymore with lotus-miner sectors remove or update-state.  Just get "ERROR: normal shutdown of state machine"

Do not seem to have any issues with other sectors though.

**Comment by jennijuju** [comment]
related #3626

**Comment by Reiers** [comment]
Hi @Meatball13 

Thanks for the report. This is fixed in the latest release.
Please upgrade Lotus.

Link to latest tags [here.](https://github.com/filecoin-project/lotus/tags)

```
v1.14.4 -  SPs should be able to force
update sectors status without getting blocked by normal shutdown of state machine.
```

Closing Lotus issue ticket.

If you are still experiencing problems, open a new ticket or ask for help in our [Discussions forum.](https://github.com/filecoin-project/lotus/discussions)
